media/static-bundles.json

```diff
@@ -1088,6 +1094,14 @@
       ],
       "name": "firefox_whatsnew_95_de"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js",
+        "js/base/mozilla-fxa-product-button.js",
+        "js/base/mozilla-fxa-product-button-init.js"
```

Comment:
If you're using `pocket_fxa_button()` then you need `mozilla-fxa-product-button` in the bundle here, but if not then you can remove these two.